scripts/jointLklDM.C

```diff
@@ -158,7 +160,9 @@ void jointLklDM(TString configFileName="$GLIKESYS/rcfiles/jointLklDM.rc",Int_t s
   else if(!channel.CompareTo("pi0pi0",TString::kIgnoreCase))     {strchannel = "#pi^{0}#pi^{0}";    minmass=0.135;}
   else if(!channel.CompareTo("gammapi0",TString::kIgnoreCase))   {strchannel = "#pi^{0}#gamma";     minmass=0.135/2.;}
   else if(!channel.CompareTo("pi0gamma",TString::kIgnoreCase))   {strchannel = "#pi^{0}#gamma";     minmass=0.135/2.;}
-  else strchannel = "";
+  else if(!channel.CompareTo("branon",TString::kIgnoreCase))     {strchannel = "branon";            minmass=0;}
+  else                                                           {strchannel = channel;             minmass=0;}
```

Comment:
channel is the string you read from the rcfile, and the one you should decode to build the dNdE histogram and to write the strchannel, that should be the latex version of channel (with greeks letters, etc) that it is used for plots. 
Please do not use strchannel for the decoding part as you do below
Now that channel can be any arbitrary string, you should write a separate function that would take channel as argument, and gives back the corresponding strchannel 
Include in the new function the option for channel="ee", that is now missing. It should be strchannel="e^{+}e^{-}"

Reply:
New commit does! Added the "ee" channel.